File: exercicios/1866/1866.py
def entrada():
    n = int(input())
    return(n)

def main():

    n = entrada()
    for i in range(n):
        v = int(input())
        if v % 2 == 0:
            print(0)
        else:
            print(1)
        i += 1

main()

File: exercicios/JavaScript/1866/1866.js
var input = require('fs').readFileSync('stdin','utf8')
var lines = input.split('\n')

var x = parseInt(lines.shift())

for(let i=0; i<x; i++){
    var v = parseInt(lines.shift())

    if(v % 2 == 0){
        console.log(0)
    }else{
        console.log(1)
    }
}